src/com/itschool/session6/homework/challenge02/Challenge02.java

```diff
@@ -0,0 +1,33 @@
+package com.itschool.session6.homework.challenge02;
+
+public class Challenge02 {
+
+    public static void main(String[] args) {
+        equalIntCheck();
+        equalStringCheck();
+}
```

Comment:
judging from this end brace one would say that here ends the class, not the `main` method. Also, the rest of the methods in this class look like they are outside the class with the current (missing) indentation. If you have Eclipse Keymap configured in IntelliJ like me, you can nicely and easy format the entire code from the class using combination CTRL + Shift + F. 
I strongly recommend formatting all the code.

Reply:
solved